frontend/src/metabase/admin/datamodel/metadata/components/MetadataTable/MetadataTable.tsx

```diff
@@ -48,7 +48,12 @@ interface StateProps {
 }
 
 interface DispatchProps {
-  onUpdateTable: (table: Table, name: string, value: unknown) => void;
+  onUpdateTable: (
+    table: Table,
+    name: string,
+    value: unknown,
+    skipMerge?: boolean,
```

Comment:
Can we not add another argument to this function?

- boolean flag arguments are a [code smell](https://www.martinfowler.com/bliki/FlagArgument.html)
- 4 arguments [is a lot](https://maximilianocontieri.com/code-smell-10-too-many-arguments) and increases complexity of all related code
- `skipMerge` name is a bit ambiguous since [`merge` function is used in `notify` & `undo` ](https://github.com/metabase/metabase/blob/ad0c5d5957bb40ecbbd156b0a5cfa19d7a3e3580/frontend/src/metabase/lib/entities.js#L661-L665)
- ideally [`notify` & `undo` functions](https://github.com/metabase/metabase/blob/ad0c5d5957bb40ecbbd156b0a5cfa19d7a3e3580/frontend/src/metabase/lib/entities.js#L661-L665) should have the same interface

To address this, the [`undo` reducer](https://github.com/metabase/metabase/blob/dca89cc3004a447ea14dd2a47683a38885c006e5/frontend/src/metabase/redux/undo.js#L52) should be refactored and related actions should become more granular. But then this small fix would turn into a sizable refactoring. On the other hand, in the current state this fix unnecessarily increases code complexity.